README.md

```diff
@@ -51,9 +51,9 @@ Auto detect binding from `v-bind:`, v-bind short hand `:m-props`, `v-model`, `v-
 ```
 
 
-## Components 
+## Vue Options 
```

Comment:
we should keep the section about component registration and add a section about `ngVueOptions` 
```suggestion
## Vue Options 

`ngVueOptions` must be object where key/value will be assigned on *Vue host wrapper component*. It allow user to add pass additional parameters to the component definition object. Eg you can use `ngVueOptions` to passe locally registered components to the vue host 

```javascript
ngVueOptions = { 
 components : { hello : MyHelloComponent }
}

Following properties are blacklisted:

- props
- data
- computed
- methods
- watch

All lifecycle hooks are not handled/tested yet :

- beforeCreate
- created
- beforeMount
- mounted
- beforeUpdate
- updated
- beforeDestroy
- destroyed
   

## Component registation
```